### PR TITLE
Composer update

### PR DESCRIPTION
**Pull request by Nicofuma**

### CODE REVIEW

**Comment by florianbelhomme** [comment]
Add RFR or WIP in your PR

**Comment by florianbelhomme** [comment]
Why update the SF version ? There is a good reason.
But be careful when you do this in real life, it can be hard to update the code to do 2.4->2.7

**Comment by Nicofuma** [comment]
I don't have the rights on the repo. Therefore I can't edit the labels. 

About Symfony, I believe its important to start woth good basis, the 2.4 version isn't supported anymore and it works well with 2.7.

**Comment by saro0h** [comment]
@Nicofuma Here you go. All the rights you need ;)

**Comment by Nicofuma** [comment]
Thanks.

**Comment by florianbelhomme** [comment]
:+1:

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ web/bundles/
 log/*
 web/css/*
 web/js/*
+bin/doctrine-migrations
```
Comment (salahm) [inline .gitignore]: this should not be commited

Comment (Nicofuma) [inline .gitignore]: I don't completely understand what do you want here:
- Commit bin/doctrine-migrations
- Add /bin/ to the .gitignore (and remove the 2 existing files?)
- Something else?

Comment (salahm) [inline .gitignore]: yes, add `/bin/*` to your `.gitignore`.

Comment (Nicofuma) [inline .gitignore]: updated

**Comment by salahm** [comment]
:+1:

**Comment by ogizanagi** [comment]
:+1:

**Pull request merged**